upup/pkg/fi/cloudup/tagbuilder.go

```diff
@@ -96,6 +96,16 @@ func buildCloudupTags(cluster *api.Cluster) (sets.String, error) {
 func buildNodeupTags(role api.InstanceGroupRole, cluster *api.Cluster, clusterTags sets.String) (sets.String, error) {
 	tags := sets.NewString()
 
+	networking := cluster.Spec.Networking
+
+	if networking == nil {
+		return nil, fmt.Errorf("Networking is not set, and should not be nil here")
+	}
+
+	if networking.LyftVPC != nil {
+		tags.Insert("_lyft_vpc_cni")
```

Comment:
You probably aren't using this tag - we're trying to get rid of tags altogether... but it's harmless

Reply:
This tag is how the cni config file template gets rendered

/upup/models/nodeup/resources/_lyft_vpc_cni/files/etc/cni/net.d/10-cni-ipvlan-vpc-k8s.conflist.template

if you can point me to a better place to handle rendering that template I could refactor accordingly